control/control.openSUSE.xml

```diff
@@ -70,6 +70,13 @@ textdomain="control"
         <!-- FATE: #303893, #305588: Default to enabled kdump -->
         <enable_kdump config:type="boolean">true</enable_kdump>
 
+        <!-- Set SELinux permissive mode by default, https://github.com/yast/yast-installation/pull/906#issuecomment-784238549 -->
+        <selinux>
+          <mode>permissive</mode>
+          <configurable config:type="boolean">true</configurable>
+          <patterns>selinux</patterns>
```

Comment:
This is the pattern found by `zypper se -s selinux`

```
➜ zypper se -s selinux

Loading repository data...
Reading installed packages...
...
| selinux                  | pattern | 20200505-11.1 | i586   | openSUSE-Tumbleweed-Oss
...
```

Reply:
As mentioned more changes are needed in skelcd. I will add it to your PR when installation is ready for it ( as it needs new yast2-installation to have new security proposal and finish client )